### DISCUSSION TITLE
Preload extension and hx-boost not rebinding preload events?

### DISCUSSION THREAD

**Discussion by derekcavaliero** [bigskysoftware/htmx#2214] [Q&A]
I feel like I am overlooking something here.

I have a simple WordPress site that I am using `hx-boost` on the `body`. Additionally - I have `preload` on the body to preload the links on the page to `mousedown` (the default). This works for the first page load, but as soon as I navigate to another page via a boosted link, I no longer see the preloading happening for the boosted links on the swapped `body`.

Am I doing something wrong here? Kinda stumped, I took a look at the source for the `preload` extension and it seems like it rebinds the events for triggering the `preload` requests via the `htmx:afterProcessNode` event hook... my HTML is very simple:

```html
<body hx-ext="head-support, preload" hx-boost="true" hx-swap="innerHTML transition:true">
  <div preload>
    <!-- page content here -->
  </div>
</body>
```

**Comment by derekcavaliero** [comment]
Basically what I am trying to achieve is preloading my boosted links on `mousedown`. The more I look at this I don't think they're compatible w/ each other.

**Comment by mdesco-crgl** [comment]
You need to call htmx.process() if what you are swapping in has htmx attributes

**Reply by derekcavaliero** [reply]
Thanks @mdesco-crgl - I'll give that a shot.
